lib/src/intl_suggestors/constants.dart

```diff
@@ -0,0 +1,17 @@
+const propsToCheck = [
```

Comment:
Is there a reason only certain props are being checked, as opposed to excluding props we don't want to check? Seems like there could be some false negatives.

Reply:
That makes sense. That reasoning might be worth capturing as a code comment somewhere.

Though, I could still see blocking props like that (just blocking `size`, `color`, `variant` might get you pretty far). Or maybe it's not so nice in practice 🤷, just thought I'd call it out.

Reply:
Yeah.  I think in either case it is a guessing game.  Guess to include user facing string, or guess to exclude non user facing strings.